README.md

```diff
@@ -126,16 +126,59 @@ The trace agent can be configured by passing a configurations object to the agen
 
 One configuration option of note is `enhancedDatabaseReporting`. Setting this option to `true` will cause database operations for redis and MongoDB to record query summaries and results as labels on reported trace spans.
 
-## Disabling the trace agent
+### Disabling the trace agent
 
 The trace agent can be turned off by either setting the `GCLOUD_TRACE_DISABLE` environment variable or specifying `enabled: false` in your configuration file.
```

Comment:
While you're here.. `GCLOUD_TRACE_DISABLE` is no more, can you remove it? Better.. get rid of this whole section.